### ISSUE TITLE
Sometimes ugly reply-quote text re-wrapping

### ISSUE THREAD

**Issue by JocelynDelalande** [mailpile/Mailpile#1264]
If I reply to a mail saying (that's a real example from a gmail's contact) :

```
Ut ut lore pas ut at feugiat ut vitae cum nisi at ut eros a sem sed ut nisi
et in vitae que feugiat pharetra Pellentesqueiii vitae.
```
It will become, once quoted :
```
> Ut ut lore pas ut at feugiat ut vitae cum nisi at ut eros a sem sed ut
> nisi
> et in vitae que feugiat pharetra Pellentesqueiii vitae.
```

Of course, 

That's because of the 72 chars line width limit.

Instead of, what one would expect :

```
> Ut ut lore pas ut at feugiat ut vitae cum nisi at ut eros a sem sed ut
> nisi et in vitae que feugiat pharetra Pellentesqueiii vitae.
```

It requires me to re-wrap text by hand, everytime.

I don't know exactly how to deal with that, it's an hard issue, as it may require some guessing, but I hope we can come with some (maybe heuristic approach ?). I guess, other mail clients should have faced that issue as well.

Any input on that ?

**Comment by BjarniRunar** [comment]
Ack, this is an issue. Not sure how best to deal with it yet. :)

**Comment by BjarniRunar** [comment]
This should be fixed by 082aec514a41e4f49880a52a155a4dd945db817a. Closing, feel free to file another or reopen if it still sucks. :-)
